### DISCUSSION TITLE
jk-bms errors what does this mean?

### DISCUSSION THREAD

**Discussion by MrYoshii** [syssi/esphome-jk-bms#141] [Q&A]
Hi, I have a question regarding the sensor jk-bms errors, the value is occasionally Error 0x08 0x00.

I have checked the log once with the jk app but there are no new errors. 
Does anyone have any ideas? 

![grafik](https://user-images.githubusercontent.com/8422439/179033973-643634ec-c9f5-4f4b-b6ab-1e2d9345d2f2.png)

**Comment by syssi** [comment]
The [implementation/decoding](https://github.com/syssi/esphome-jk-bms/blob/main/components/jk_bms_ble/jk_bms_ble.cpp#L21-L38) of the error bitmask is incomplete and will be improved in future:

https://github.com/syssi/esphome-jk-bms/issues/115

I've extracted some error messages from the JK-BMS app but I don't know the corresponding bits yet. I'm trying to implement a JK-BMS emulator at the moment to send the different error bits to the app and see what's reported.
